README.md

```diff
@@ -500,6 +525,7 @@ def living_room():
   following 2 arguments:
   * `given_that` _- For configuring the state_
   * `hass_functions` _- For more complex setup steps_
```

Comment:
Could you replace `hass_functions` with `hass_mocks` in this line?

> following 2 arguments:
>  * `given_that` _- For configuring the state_
>  * `hass_mocks` _- For more complex setup steps_
>    * Note: `hass_functions` was deprecated in favor of using `hass_mocks`

Reply:
all set.

Reply:
👍